library/src/main/java/com/billing/amazon/AmazonBillingListener.java

```diff
@@ -74,15 +74,15 @@ public void onPurchaseResponse(PurchaseResponse response) {
                     Log.d(TAG, "onPurchaseResponse: receipt json:" + receipt.toJSON());
                     Log.d(TAG, "onPurchaseResponse: getUserId:" + response.getUserData().getUserId());
                     Log.d(TAG, "onPurchaseResponse: getMarketplace:" + response.getUserData().getMarketplace());
```

Comment:
What do you think, is it good idea to log debug messages only if BuildConfig.DEBUG is true?